#!/usr/bin/env python2.7

PACKAGE='hanse_pingerfollowing'
import roslib; roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()


gen.add("init_step", double_t, 0, "", 2, -1000, 1000)
gen.add("step", double_t, 0, "", 0.1, -1000, 1000)
gen.add("bias_step", double_t, 0, "", 1, -1000, 1000)
gen.add("mu", double_t, 0, "", 0.5, 0, 1)
gen.add("hydrophone_direction", double_t, 0, "", 0.5, 0, 1)
gen.add("bias", double_t, 0, "", 0.5, 0, 0.5)
gen.add("angular_speed", double_t, 0, "", 0.75, 0, 1.0)
gen.add("turn_time", double_t, 0, "", 0.75, 0, 1.0)
gen.add("linear_speed", double_t, 0, "", 0.75, 0, 1.0)
gen.add("min_one_step", bool_t, 0, "", True)

exit(gen.generate(PACKAGE, "hanse_pingerfollowing", "PingerFollowing"))
